source/conf.py

```diff
@@ -16,6 +16,8 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 
+import sphinx_rtd_theme
```

Comment:
This is the way described in the documentation https://github.com/snide/sphinx_rtd_theme#via-package

If you look at line 89 in the diff you'll see that this import is actually called. I agree that it's kinda weird.